src/services/logging/default-logger.ts

```diff
@@ -1,29 +1,39 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { ToStringHelper } from './tostring-helper';
+
 /**
  * The default logger performs toString() operations on each param passed to the logging actions
  */
 export class DefaultLogger {
+  toString = (m?: any): any => {
```

Comment:
This does only perform toString on part of the code, so not sure that's the appropriate name.